src/modules/AutoinstPartPlan.ycp

```diff
@@ -359,7 +359,21 @@
         StorageDevices::InitDone();
         map<string, map> StorageMap = eval(Storage::GetTargetMap());
 
-        StorageMap=filter(string d, map p, StorageMap, ``( d != "/dev/evms" && d != "/dev/nfs" && d != "/dev/tmpfs" && size(p["partitions"]:[]) > 0));
+        StorageMap=filter(string d, map p, StorageMap, ``{
+	  boolean ok = (d != "/dev/evms" && d != "/dev/tmpfs" && size(p["partitions"]:[]) > 0);
+          if( ok && d == "/dev/nfs" ){
+            // Checking if /dev/nfs container has a root partition.
+            // If yes, it can be taken for the plan (bnc#986124)
+            ok = false;
+            foreach(map part, p["partitions"]:[], ``{
+		if( part["mount"]:"" == "/" ){
+                ok = true;
+                break;
+              }
+            });
+	  }
```

Comment:
Please fix the indentation. Use a C/C++ mode in the editor if you no longer have a YCP mode.